README.md

```diff
@@ -307,7 +307,7 @@ The SDK supports end-to-end encryption via the Olm and Megolm protocols, using
 [libolm](https://gitlab.matrix.org/matrix-org/olm). It is left up to the
```

Comment:
@robertlong Is this still relevant to merge? It looks like it was originally created to address https://github.com/vector-im/element-call/issues/257 which was closed in a different fashion.